src/Trixi.jl

```diff
@@ -222,6 +222,8 @@ export ncomponents, eachcomponent
 
 export TreeMesh, StructuredMesh, UnstructuredMesh2D, P4estMesh, T8codeMesh
 
+export adapt!
+
```

Comment:
I'm not sure whether we really want to `export` such a common name. Could we instead just use `Trixi.adapt!` in the elixirs?

Reply:
Refactored as you suggested.